src/components/pages/Dashboard.jsx

```diff
@@ -1,33 +1,48 @@
-import { logout } from "../../util/DataBaseRequests";
+import { Outlet, useLocation } from "react-router-dom";
+import { useEffect, useState } from "react";
+
+import DashboardNav from "../navbars/DashboardNav";
+import { Home } from "./dashboard-pages/index";
+import { getUserData } from "../../util/DataBaseRequests";
 import { useAuth } from "../../AuthProvider";
-import { useNavigate } from "react-router-dom";
-import { useState } from "react";
 
 const Dashboard = () => {
-    // temporarily this functionality is here, then it will be moved to the header
-    const [logoutError, setLogoutError] = useState({});
-    const { clearUserSession, userData } = useAuth();
-    
-    const navigate = useNavigate();
-    const handleLogout = async () => {
+    const location = useLocation();
+    const [profile, setProfile] = useState({});
+    const [error, setError] = useState({});
+    const { userData, clearUserSession } = useAuth();
+
+    useEffect(() => {
+        console.log('useeffect')
+        const fetchUserProfileData = async () => {
         try {
-            const result = await logout();
-            if (result.status === 200){
-                clearUserSession();
-                navigate('/');
+            const result = await getUserData(userData._id, userData.token);
+            if (result.status === 200) {
+                setProfile({
+                    firstName: result.data.firstName,
+                    lastName: result.data.lastName,
+                    email: result.data.email,
+                    dateOfBirth: new Date(result.data.dateOfBirth).toLocaleString("en-US", { timeZone: 'UTC', year: 'numeric', month: 'long', day: 'numeric' })|| "",
+                    adultName: result.data.adultName || "",
+                    role: result.data.role,
+                });
+                setError('');
```

Comment:
Can the default message be set to and empty string to eliminate the need to set it to empty?